server/routes/pageFlowHistory.ts

```diff
@@ -1,10 +1,23 @@
+import { Request } from 'express'
 import type { PageFlow } from 'viewModels'
 
 /**
  * Module exposing utility functions that operate on [PageFlow] instances. This is to maintain a history of pages
  * that have been visited as part of a journey (e.g. for back links on forms).
  */
 
+/**
+ * Builds and returns a new [PageFlow] instance to be used as a Page Flow History, using the current route as the
+ * initial page in the history.
+ */
+const buildNewPageFlowHistory = (req: Request): PageFlow => {
```

Comment:
This is the new common method, which is basically the same as all the individual ones that this PR removes 👍